src/test/java/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,30 @@
+package impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import org.junit.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class FileReaderImplTest {
+    private static final String PATH = "src/test/java/TestsFiles/TestInputData.csv";
+
+    @Test
+    public void reader_Test_Ok() {
```

Comment:
be consistent in naming, always use lowercase after underscore
first part is name of method under test
second part of the name describes input, arguments, conditions, what you are passing to the tested function
```suggestion
    public void readFile_existingFile_ok() {
```